internal/config/config.go

```diff
@@ -11,6 +11,9 @@ type Config struct {
 	Port                    int
 	SendgridAPIKey          string
 	SlackAPIKey             string
+	TwilioAccID             string
```

Comment:
Can you change this to `TwilioAccountID` wherever it's used if "Acc" is short for "Account"?

Reply:
@bmonkman okay to leave the `env` variables as `TWILIO_ACC_ID` or should that be updated as well?